lightly/models/batchnorm.py

```diff
@@ -21,15 +25,18 @@ class SplitBatchNorm(nn.BatchNorm2d):
 
     """
 
-    def __init__(self, num_features, num_splits, **kw):
+    running_mean: torch.Tensor
+    running_var: torch.Tensor
```

Comment:
I think this should be moved to the `__init__` method as those are not class variables.

Reply:
Unfortunately it doesn't work if I move it to `__init__`. I have reverted to using type ignores for now.

Reply:
This is weird, I tried it and it worked for me.